django/core/management/commands/showmigrations.py

```diff
@@ -26,7 +26,11 @@ def add_arguments(self, parser):
         )
         formats.add_argument(
             '--plan', '-p', action='store_const', dest='format', const='plan',
-            help='Shows all migrations in the order they will be applied.',
+            help=(
+                'Shows all migrations in the order they will be applied. '
```

Comment:
So, it seems like you changed behavior in addition to simply making refactors. It would be idea if you could split this into several commits to make that more clear and to make review easier.